pymc3/distributions/continuous.py

```diff
@@ -194,7 +210,7 @@ def logp(self, value):
                      tau > 0, sd > 0)
 
 
-class HalfNormal(PositiveContinuous):
+class HalfNormal(PositiveUnivariateContinuous):
```

Comment:
Started splitting those up in the most recent commit; however, it became apparent that constructor call order, and other confusing things, could arise in this scenario.